cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -68,12 +68,17 @@
     /sys/fs/cgroup/freezer/snap.*/ w,
     /sys/fs/cgroup/freezer/snap.*/cgroup.procs rw,
 
-    # cgroup: pids
+    # cgroup: snapd
+    # allow creating and configuring the snapd hierarchy
+    /run/snapd/ rw,
+    /run/snapd/cgroup/ rw,
```

Comment:
I think these just need 'w' based on the above. Is 'r' needed for the next PR?

Reply:
I don't think `r` is needed but I'll double check.

Reply:
I checked and reduced the permissions as suggested.